backend/src/Service/Order/OrderService.php

```diff
@@ -9,12 +9,14 @@
 use App\Response\Order\OrderResponse;
 use App\Response\Order\OrdersResponse;
 use App\Response\Subscription\CanCreateOrderResponse;
+use App\Constant\Notification\NotificationConstant;
 use App\Constant\Subscription\SubscriptionConstant;
 use App\Service\Subscription\SubscriptionService;
+use App\Service\Notification\NotificationLocalService;
 
 class OrderService
 {
-    public function __construct(private AutoMapping $autoMapping, private OrderManager $orderManager, private SubscriptionService $subscriptionService)
+    public function __construct(private AutoMapping $autoMapping, private OrderManager $orderManager, private SubscriptionService $subscriptionService, private NotificationLocalService $notificationLocalService)
```

Comment:
We need to define the class properties outside the constructor function and initialize them inside the constructor body.